softwareComponents/simplesimClientLib/include/changecolor.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+//#include "simplesim_client.hpp"
+
+#include <QListWidgetItem>
+
+namespace Ui {
+    class ChangeColor;
+}
+
+namespace rofi::simplesim
+{
+
+struct ChangeColor : public QWidget
+{
+    Q_OBJECT
+
+    Ui::ChangeColor* ui;
```

Comment:
Couldn't you use `std::unique_ptr` here and save all the trouble with constructors and destructors?

Reply:
Seems to work fine with unique_ptr